roles/confluent.control_center/tasks/main.yml

```diff
@@ -54,6 +58,17 @@
     group: "{{control_center_group}}"
     recurse: true
 
+# Archive File deployments need to create SystemD service units
+# Copy the tarball's systemd service to the system
+- name: Copy Control Center Service from archive file to system
+  copy:
+    src: "{{binary_base_path}}/lib/systemd/system/{{control_center.systemd_file|basename}}"
+    remote_src: true
+    dest: "{{control_center.systemd_file}}"
+    mode: 0644
+    force: "{{overwrite_systemd_services|bool}}"
```

Comment:
wondering what the value of making the force toggleable is? Like if someone sets this to false then wouldnt that lead to deployment issues?

Reply:
Agreed. I had it as a toggle in case the user didn't want to modify system files. This can be removed or changed to the `backup` property if you think that is appropriate.

Just let me know how you want to proceed.

Reply:
Let's remove the toggle please. Likely this systemd file will be exactly what they got if they installed the package anyhow

Reply:
Fixed in commit 30c84953